modules/apps/portal-remote/portal-remote-soap-extender-api/src/main/java/com/liferay/portal/remote/soap/extender/SoapDescriptorBuilder.java

```diff
@@ -18,9 +18,12 @@
 
 import javax.xml.namespace.QName;
 
+import org.osgi.annotation.versioning.ProviderType;
+
 /**
  * @author Carlos Sierra Andrés
  */
+@ProviderType
```

Comment:
@W2jx Do not remove `@ProviderType`. This actually means the implementation/service should be provided by Liferay.